Message-Id: <f96b7f473f61a8e6d4866a49d9fcc861bed443b5.1336068799.git.agx@sigxcpu.org>
In-Reply-To: <308046ae1d09cc7cce80fdb268cfb7fee35de13d.1336068799.git.agx@sigxcpu.org>
References: <5909887366833c92178e3ed650b87d666f9f50c1.1336068799.git.agx@sigxcpu.org>
	<56db9d711b7615ece3848c57f71f362f0e5f4fea.1336068799.git.agx@sigxcpu.org>
	<727a1c60fdc68e24b2f457f5b7e3611b2aae7a7b.1336068799.git.agx@sigxcpu.org>
	<af426bb880140c2941366a3a84fd11a714b9c778.1336068799.git.agx@sigxcpu.org>
	<64360f6da8f2d82618e0eda5d971ee6a083851ae.1336068799.git.agx@sigxcpu.org>
	<308046ae1d09cc7cce80fdb268cfb7fee35de13d.1336068799.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Apr 2011 10:37:15 +0200
Subject: Brand fr

---
 fr/locale/fr/lightning/lightning.dtd        |    2 +-
 fr/locale/fr/lightning/lightning.properties |    6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fr/locale/fr/lightning/lightning.dtd b/fr/locale/fr/lightning/lightning.dtd
index 990a846..32b8997 100644
--- a/fr/locale/fr/lightning/lightning.dtd
+++ b/fr/locale/fr/lightning/lightning.dtd
@@ -46,7 +46,7 @@
   -->
 
 <!-- Tools menu -->
-<!ENTITY lightning.taskLabel "Lightning">
+<!ENTITY lightning.taskLabel "Iceowl">
 
 <!-- New menu popup in File menu -->
 <!ENTITY lightning.menupopup.new.event.label    "Événement…">
diff --git a/fr/locale/fr/lightning/lightning.properties b/fr/locale/fr/lightning/lightning.properties
index d0f6645..abcb753 100644
--- a/fr/locale/fr/lightning/lightning.properties
+++ b/fr/locale/fr/lightning/lightning.properties
@@ -38,12 +38,12 @@
 # ***** END LICENSE BLOCK *****
 
 # addon description:
-extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Lightning
+extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.name=Iceowl
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.description=Un agenda intégré pour Thunderbird
 extensions.{e2fda1a4-762b-4020-b5ad-a41df1933103}.creator=Projet Mozilla Calendar
 
 # Lightning branding
-brandShortName=Lightning
+brandShortName=Iceowl
 
 # Task mode title
 taskModeApplicationTitle=Tâches
@@ -72,7 +72,7 @@ imipBarRequestText=Ce message contient une invitation.
 imipBarUpdateText=Ce message contient une mise à jour pour un événement existant.
 imipBarAlreadyProcessedText=Ce message contient un événement qui a déjà été traité.
 imipBarReplyText=Ce message contient une réponse à une invitation.
-imipBarUnsupportedText=Ce message contient un événement que cette version de Lightning ne peut pas traiter.
+imipBarUnsupportedText=Ce message contient un événement que cette version de Iceowl ne peut pas traiter.
 imipBarProcessingFailed=Le traitement du message a échoué. État : %1$S.
 imipAcceptInvitation.label=Accepter
 imipCancelInvitation.label=Supprimer
